tests/api.c

```diff
@@ -24171,7 +24175,9 @@ static int test_wc_ecc_signVerify_hash (void)
 {
     int         ret = 0;
 
-#if defined(HAVE_ECC) && defined(HAVE_ECC_SIGN) && !defined(NO_ASN) && !defined(WC_NO_RNG)
+#if defined(HAVE_ECC) && defined(HAVE_ECC_SIGN) && !defined(NO_ASN) && \
```

Comment:
Line length, please limit to 80 characters. Use `\` to extend macro expression to new line.

Reply:
Reverted changes by assuming `--disable-examples`